GUI: Re-order config settings

RedDevilus opened the pull request:
### Description of Changes
<!-- Brief description or overview on what was changed in the PR -->
Re-order config setting to older order.

Master:
![image](https://user-images.githubusercontent.com/24227051/124399087-90e85c00-dd19-11eb-96f5-934f434b20f5.png)

PR:
![image](https://user-images.githubusercontent.com/24227051/124399096-9c3b8780-dd19-11eb-9986-1611d1e1241c.png)


### Rationale behind Changes
<!-- Why were these changes made?  What problem does it solve / area does it improve? -->
Visual clarity between option and also muscle memory.
### Suggested Testing Steps
<!-- If applicable, including examples you've already tested with / recommendations for how to test further is very helpful! -->

arcum42 commented:
Seriously, the whole thing is a mess anyways. Eight separate setting dialog boxes in eight different menu items? And how many times can we use the word "settings" in a menu?

We really need to combine a lot of these...

RedDevilus commented:
Yeah, a poll would help in this case.

arcum42 commented:
If we were just going to do some general renaming, I'd suggest something like:
General...
Memory cards...
Bios...
--
Graphics...
Sound...
Gamepads...
Network/Hard drive...
Usb...
--
Multitap 1
Multitap 2
--
Language...
Reset to defaults

Though I'm sure all the translations would break. Language *could* just be a menu, too...

RedDevilus commented:
Languages are held back anyway, so by anything they are already broken. So don't hold back if you have suggestions.

Shoegzer commented:
@arcum42 Out of curiosity, are all the ellipses necessary?  Just seems cleaner without them.

arcum42 commented:
> @arcum42 Out of curiosity, are all the ellipses necessary? Just seems cleaner without them.

It's a standard ui convention that if a menu item opens a dialog box, it has an ellipse after it, and all of those open dialog boxes.

tellowkrinkle commented:
While you're at it, maybe move the `&` on `GamePad Settings` to `Game&Pad Settings` to reduce conflict?

> Though I'm sure all the translations would break.

There aren't any to break
<img width="204" alt="screenshot broken translations" src="https://user-images.githubusercontent.com/3315070/124536783-395ff400-ddde-11eb-9318-3ff3795469ff.png">

RedDevilus commented:
![image](https://user-images.githubusercontent.com/24227051/124841538-c4cbb780-df8d-11eb-92d9-02c4d4acffd1.png)

Macro is fine now as well.

RedDevilus commented:
![image](https://user-images.githubusercontent.com/24227051/125135830-f073ac80-e109-11eb-8c3d-6bd022954e22.png)
Removed the punctations marks/ dots.

RedDevilus commented:
![image](https://user-images.githubusercontent.com/24227051/125139144-4cd9ca80-e110-11eb-9a0c-d9f682458b7e.png)
Added settings back and removed the dots for about.

RedDevilus commented:
![4446 Current Design](https://user-images.githubusercontent.com/24227051/126042572-2aa3893e-3fd8-4ba3-abeb-70fa4c1a6c29.gif)
Hopefully this is fine unless someone has some more input. It should be ready for merging.

lightningterror commented on pcsx2/gui/AppRes.cpp:
```diff
@@ -42,7 +42,7 @@
 {
 	Menu = std::unique_ptr<wxMenu>(new wxMenu());
-	Menu->Append(MenuId_IsoBrowse, _("Browse..."), _("Browse for an ISO that is not in your recent history."));
+	Menu->Append(MenuId_IsoBrowse, _("Browse"), _("Browse for an ISO that is not in your recent history."));
```
The dots should stay here.

lightningterror commented on pcsx2/gui/MainFrame.cpp:
```diff
@@ -61,7 +61,7 @@ wxMenu* MainEmuFrame::MakeStatesSubMenu(int baseid, int loadBackupId) const
 	// For safety i also made them inactive aka grayed out to signify that's it's only for informational purposes
 	// Fixme: In the future this can still be expanded to actually cycle savestates in the GUI.
-	mnuSubstates->Append(baseid - 1, _("File..."));
+	mnuSubstates->Append(baseid - 1, _("File"));
```
Here too.

lightningterror commented on pcsx2/gui/MainFrame.cpp:
```diff
@@ -374,7 +374,7 @@ void MainEmuFrame::CreatePcsx2Menu()
 		_("Wipes all internal VM states."));
 	m_menuSys.FindItem(MenuId_Sys_Shutdown)->Enable(false);
 
-	m_menuSys.Append(MenuId_Boot_ELF, _("&Run ELF..."),
```
Ditto.

lightningterror commented on pcsx2/gui/MainFrame.cpp:
```diff
@@ -506,7 +507,7 @@ void MainEmuFrame::CreateCaptureMenu()
 	keyCode.Cmd(false);
 	AppendShortcutToMenuOption(*sysScreenShotItem, keyCode.toTitleizedString());
-	m_submenuScreenshot.Append(MenuId_Capture_Screenshot_Screenshot_As, _("Screenshot As..."));
+	m_submenuScreenshot.Append(MenuId_Capture_Screenshot_Screenshot_As, _("Take Screenshot And Browse"));
```
Save Screenshot As... ?
